release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -750,10 +754,19 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-management-console-bug-fixes"]
 ==== Management Console
 
+* Previously, a terminated container would render as a meaningless string in the internationalization code results. For example, the `{{label}}` and {{exitCode}}' codes would display in the results with no human-readable string output, such as `Terminated with exit code 13` for each terminated container. The {product-title} {product-version} fixes this issue with an update to the internationalization code that now ensures a human-readable string exists in the internationalization code results for a container that was terminated. (link:https://issues.redhat.com/browse/OCPBUGS-4206[*OCPBUGS-4206])
```

Comment:
Feel free to keep yours, but you could cut it down to follow some of the minimalism guidelines.

```suggestion
* Previously, a terminated container would render `{{label}}` and {{exitCode}}' codes  for each terminated container. With this update,  the internationalization code is fixed to render a readable output message.(link:https://issues.redhat.com/browse/OCPBUGS-4206[*OCPBUGS-4206])
```